lib/charms/hpc_libs/v0/slurm_ops.py

```diff
@@ -163,17 +129,34 @@ def _snap(*args) -> str:
     return _call("snap", *args)
 
 
-def _mungectl(*args: str, stdin: Optional[str] = None) -> str:
-    """Control munge via `slurm.mungectl ...`.
+def format_key(key: str) -> str:
```

Comment:
Are we still using this method anywhere? Quick search for usages seems to indicate that we're no longer using it except within the unit tests. Side effect of not letting the snap dictate everything.